build.gradle

```diff
@@ -24,6 +24,7 @@ buildscript {
     dependencies {
         classpath 'com.netflix.nebula:gradle-netflixoss-project-plugin:9.4.2'
         classpath 'io.mantisrx:mantis-gradle-plugin:1.2.+'
+        classpath "io.freefair.gradle:lombok-plugin:5.3.3.3"
```

Comment:
Instead of using a plugin, can we just use a compileOnly dependency. See mantis-example-core/build.gradle for example.

Reply:
The plugin seems to be the recommended method. Can you share your concerns here?

Reply:
I don't want to maintain 2 different convention of pulling in a library within the same code base. I think the consistency helps reduce confusion in the future.

Reply:
I tried removing the old ref in the example folder and it seems to be working fine in the build.

Reply:
@calvin681 @nickmahilani I did the runtime dependency comparison between this build and the master and there is no runtime dependency introduced by lombok. (I tried common and runtime folder.)

```
runtimeClasspath - Runtime classpath of source set 'main'.
+--- project :mantis-remote-observable
|    +--- project :mantis-common
|    |    +--- project :mantis-shaded
|    |    +--- com.netflix:mantis-rxnetty:0.4.19.1
|    |    |    +--- io.reactivex:rxjava:1.0.17 -> 1.3.8
|    |    |    +--- io.netty:netty-handler:4.1.5.Final -> 4.1.17.Final
|    |    |    |    +--- io.netty:netty-buffer:4.1.17.Final
|    |    |    |    |    \--- io.netty:netty-common:4.1.17.Final
|    |    |    |    +--- io.netty:netty-transport:4.1.17.Final
|    |    |    |    |    +--- io.netty:netty-buffer:4.1.17.Final (*)
|    |    |    |    |    \--- io.netty:netty-resolver:4.1.17.Final
|    |    |    |    |         \--- io.netty:netty-common:4.1.17.Final
|    |    |    |    \--- io.netty:netty-codec:4.1.17.Final
|    |    |    |         \--- io.netty:netty-transport:4.1.17.Final (*)
|    |    |    +--- io.netty:netty-codec-http:4.1.5.Final -> 4.1.17.Final
|    |    |    |    \--- io.netty:netty-codec:4.1.17.Final (*)
|    |    |    +--- io.netty:netty-transport-native-epoll:4.1.5.Final -> 4.1.17.Final
|    |    |    |    +--- io.netty:netty-common:4.1.17.Final
|    |    |    |    +--- io.netty:netty-buffer:4.1.17.Final (*)
|    |    |    |    +--- io.netty:netty-transport-native-unix-common:4.1.17.Final
|    |    |    |    |    +--- io.netty:netty-common:4.1.17.Final
|    |    |    |    |    \--- io.netty:netty-transport:4.1.17.Final (*)
|    |    |    |    \--- io.netty:netty-transport:4.1.17.Final (*)
|    |    |    \--- org.slf4j:slf4j-api:1.7.6 -> 1.7.32
|    |    +--- io.reactivex:rxjava:1.3.8
|    |    +--- io.netty:netty-codec-http:4.1.17.Final (*)
|    |    +--- io.netty:netty-buffer:4.1.17.Final (*)
|    |    +--- io.netty:netty-transport-native-epoll:4.1.17.Final (*)
|    |    +--- com.netflix.numerus:numerus:1.+ -> 1.1
|    |    +--- org.xerial.snappy:snappy-java:1.1.+ -> 1.1.8.4
|    |    +--- org.jctools:jctools-core:1.+ -> 1.2.1
|    |    +--- com.netflix.spectator:spectator-api:0.92.+ -> 0.92.2
|    |    |    \--- org.slf4j:slf4j-api:1.7.26 -> 1.7.32
|    |    +--- org.slf4j:slf4j-api:1.7.+ -> 1.7.32
|    |    \--- org.slf4j:slf4j-log4j12:1.7.+ -> 1.7.32
|    |         +--- org.slf4j:slf4j-api:1.7.32
|    |         \--- log4j:log4j:1.2.17
|    +--- io.netty:netty-handler:4.1.17.Final (*)
|    +--- com.netflix.rxjava:rxjava-math:0.20.6
|    +--- org.slf4j:slf4j-api:1.7.+ -> 1.7.32
|    \--- org.slf4j:slf4j-log4j12:1.7.+ -> 1.7.32 (*)
+--- project :mantis-network
|    +--- io.netty:netty-handler:4.1.17.Final (*)
|    +--- io.mantisrx:mql-jvm:3.2.+ -> 3.2.9
|    |    \--- io.reactivex:rxjava:1.3.8
|    \--- project :mantis-common (*)
+--- project :mantis-common (*)
\--- org.slf4j:slf4j-api:1.7.+ -> 1.7.32
```